pythran/run.py

```diff
@@ -82,6 +84,9 @@ def run():
                         'generation',
                         default=list())
 
+    parser.add_argument('-C', dest='package', type=str,
```

Comment:
That's a strange option name; Maybe only accept long option?

Reply:
sure